rmf.repos

```diff
@@ -70,4 +70,4 @@ repositories:
   thirdparty/ros_ign:
     type: git
     url: https://github.com/ignitionrobotics/ros_ign.git
-    version: ros2
+    version: galactic
```

Comment:
This is not the right fix for us, since we want to target `fortress`. More information [here](https://github.com/open-rmf/rmf_simulation/pull/70). We will probably need to fix the failure on our end but continue using `ros2` branch

Reply:
Jut tested it to be working with `fortress`, thanks!